recipes/libsvm/3.16/build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 make
-mv svm-train $PREFIX
-mv svm-scale $PREFIX
-mv svm-predict $PREFIX
+mkdir $PREFIX/bin
```

Comment:
I can try without, but mkdir without `-p` should fail if there is already a directory, isn't it?